packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx

```diff
@@ -23,6 +23,7 @@ const DeviceAcquire = () => {
                 label={<Translation id="TR_ACQUIRE_DEVICE_TITLE" />}
                 cta={
                     <Button
+                        data-test="@omboarding-device-acquire"
```

Comment:
Typo `omboarding` -> `onboarding`.

Reply:
```suggestion
                        data-test="@onboarding/device-acquire"
```
or 
```suggestion
                        data-test="@device-acquire"
```
not sure if the initial screen is really "onboarding"

Reply:
yes. better to follow the pattern set here https://github.com/trezor/trezor-suite/blob/b6e27b1d8243ff7c9a5aab9bd1e92bd117339bcb/packages/suite/src/views/suite/device-seedless/index.tsx#L8 for all components within "PrerequisitesGuide" folder

Reply:
534889de5464e01a3eb5970394719506dc079b1e done

Reply:
@mroz22 so what @wendys-cats did is fine right?

Reply:
I think it is acceptable.